ts/components/ui/ButtonSolid.tsx

```diff
@@ -15,14 +15,13 @@ import Animated, {
   Extrapolate,
   interpolateColor
 } from "react-native-reanimated";
-import themeVariables from "../../theme/variables";
 import { IOColors, IOColorType } from "../core/variables/IOColors";
 import { IOSpringValues, IOScaleValues } from "../core/variables/IOAnimations";
 import { BaseTypography } from "../core/typography/BaseTypography";
 import { IOButtonStyles } from "../core/variables/IOStyles";
 import { WithTestID } from "../../types/WithTestID";
 
-type Props = WithTestID<{
+export type ButtonSolid = WithTestID<{
```

Comment:
Won't it be confusing to have the same name to define type and component?

Anyway do we need to export the type of its props definition?

(Same comment is valid for `ButtonOutline` too)